assets/quick-order-form.css

```diff
@@ -280,3 +280,47 @@
     padding-top: 1.4rem;
   }
 }
+
+/* Sticky header */
+
+.quick-order-form .quick-order-list__table--sticky thead {
+  position: sticky;
+  top: 0;
+  z-index: 3; /* Prevent volume pricing popover overlap */
+  background-color: rgb(var(--color-background));
```

Comment:
Hey Andrew. It seems you permanently change the `background-color` of the `thead`.   The `background-color` of the `thead` is already declared [here](https://github.com/Shopify/dawn/pull/3132/files#diff-64ed996b357ba32d262a457433275b24c99d4b5a6d3228115c64d72a026a9f4eR9-R11) and I think we want to keep this color until the sticky header is triggered. When it's triggered we need to change it to the color you declare here, then as we scroll up back we need to return the original color.